src/integration/pages/homePage/arabic/__snapshots__/canonical.test.js.snap

```diff
@@ -216,7 +216,8 @@ exports[`Canonical Home Page Page content First item in the first curation is th
 <img
   alt="الرياضه"
   class="bbc-139onq"
-  loading="lazy"
+  fetchpriority="high"
```

Comment:
Where has fetchpriority come from (I didn't see any references in this PR)?

Reply:
was there anyway, just not being called

https://github.com/bbc/simorgh/blob/bf667c729384a45eaee0718ddc9af8710614db8f/src/app/components/Curation/HierarchicalGrid/index.tsx#L63-L64

part of this work https://github.com/bbc/simorgh/commit/bf667c729384a45eaee0718ddc9af8710614db8f